com.unity.formats.alembic/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [2.3.1] - 2022-03-22
+## [2.3.2] - 2022-03-22
```

Comment:
Maybe we can use the occasion to update the date a bit.

Reply:
updated to todays date!